pkgs/games/pegasus-frontend/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+, cmake
+, libsForQt5
+, wrapQtAppsHook
+, SDL2
+, sqlite
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pegasus-frontend";
+  version = "22-05-2023-6421d7a";
```

Comment:
Please follow the version format from [package naming section of the manual](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming).

Reply:
done